src/Modules/Internationalisation/templates/Backend/Actions/TranslationIndex.html.twig

```diff
@@ -58,12 +58,12 @@
   {% if notFiltered is defined %}
     {{ pageon_datagrid(notFiltered) }}
   {% elseif dataGrids is defined %}
-    {% for dataGrid in dataGrids %}
+    {% for data_grid in dataGrids %}
```

Comment:
Shouldn't `dataGrids` be changed to `data_grids`?

Reply:
Don't forget to update the php as well

Reply:
fixed 25b6728f51f9b66b32727491083760956a91456b